addons/web/static/tests/views/fields/image_field.test.js

```diff
@@ -761,3 +761,42 @@ test("unique in url does not change on record change if reload option is set to
     await clickSave();
     expect(getUnique(queryFirst(".o_field_image img"))).toBe("1659688620000");
 });
+
+test("convert image to webp", async () => {
+    onRpc(({ method, model, args }) => {
+        if(method == "create_unique" && model == "ir.attachment") {
+            // This RPC call is done two times - once for storing webp and once for storing jpeg
```

Comment:
why twice? Looks like a bug...

Reply:
It is not a bug. It is making the first ORM call from [here](https://github.com/odoo/odoo/blob/097f6c8709e4cbecf8d08e3683225b76fde97f8f/addons/web/static/src/views/fields/image/image_field.js#L167) and the other one from [here](https://github.com/odoo/odoo/blob/097f6c8709e4cbecf8d08e3683225b76fde97f8f/addons/web/static/src/views/fields/image/image_field.js#L183)